src/libai/nn/supervised/MLP.java

```diff
@@ -201,10 +201,10 @@ public void train(Matrix[] patterns, Matrix[] answers, double alpha, int epochs,
 	private void standardBP(Matrix[] patterns, Matrix[] answers, double alpha, int epochs, int offset, int length, double minerror) {
 		int[] sort = new int[length];
 		double error = error(patterns, answers, offset, length);
-		Matrix e = new Matrix(answers[0].getRows(), answers[0].getColumns());
```

Comment:
same as above, this is just to know the size of the vector to be created, it doesn't have anything to do with the offset honoring